### PR TITLE
vagrant: create known_hosts file if it does not exist

### PR DESCRIPTION
**Pull request by sxa**
Fixes this problem which occurs if there is no known_hosts file:
```
13:34:03 ++ cat playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
13:34:03 + ssh-keygen -R '[127.0.0.1]:2200'
13:34:03 do_known_hosts: hostkeys_foreach failed: No such file or directory
13:34:04 Build step 'Execute shell' marked build as failure
13:34:05 [Checks API] No suitable checks publisher found.
13:34:05 Finished: FAILURE
```
Testing under https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/896/

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

File: ansible/pbTestScripts/vagrantPlaybookCheck.sh [hunk 1]
```diff
@@ -232,6 +232,8 @@ startVMPlaybook()
 	rm -f playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
 	echo "[127.0.0.1]:${vagrantPORT}" >> playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx
 	# Remove IP from known_hosts if already found
+	# ssh-keygen -R will fail if the known_hosts file does not exist
+        [ ! -r $HOME/.ssh/known_hosts ] && touch $HOME/.ssh/known_hosts && chmod 644 $HOME/.ssh/known_hosts
```
Comment (Willsparker) [inline ansible/pbTestScripts/vagrantPlaybookCheck.sh:236]:
```suggestion
	[ ! -r $HOME/.ssh/known_hosts ] && touch $HOME/.ssh/known_hosts && chmod 644 $HOME/.ssh/known_hosts
```

**Review by Willsparker — CHANGES_REQUESTED** [review]

**Comment by sxa** [comment]
[Failing run](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/896/OS=Debian10,label=vagrant/console)
[Passing run with this fix](https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/OS=Debian10,label=vagrant/898/console)

@Willsparker What did you to do put the tab character into the suggestion?

**Comment by Willsparker** [comment]
@sxa copied and pasted it from the line above the one that needed to be made :sweat_smile: . Not being able to type it in GitHub really annoys me

**Review by Willsparker — APPROVED** [review]
LGTM- VPC run has started as expected

**Pull request merged**